.release-please-manifest.json

```diff
@@ -0,0 +1,134 @@
+{
+  "src/apis/abusiveexperiencereport": "0.1.0",
```

Comment:
Huh - with `package.json` files available in sub-directories, I wonder what the relationship is between this version and the one there?  Or is this a `semantic-release` type situation where the version in `package.json` is ignored?

Reply:
The manifest based approach that has been developed treats the manifest as ground truth -- partially because it makes it easier to support many languages, without knowing how to lookup version for every single file for mat (_package.json_, _GemFile_, _setup.py_, etc).

Reply:
Got it.   If that's the case, please change the generator to blank the version number in the deep nested `package.json` files we generate.  In my semantic-release based repos, I set it to `0.0.0` just to signal to folks `aaaaarrrrrgh, here be dragons`.